avocado/build/Dockerfile

```diff
@@ -5,11 +5,10 @@ MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
 WORKDIR /home
 
 # clone avocado
-RUN git clone https://github.com/fnothaft/avocado.git
+RUN git clone https://github.com/bigdatagenomics/avocado.git
```

Comment:
@fnothaft if you could yea or nay this, I needed to update the missing branch to pass the build.